RInAppMessaging/Classes/InAppMessagingModule.swift

```diff
@@ -109,6 +118,16 @@ internal class InAppMessagingModule: AnalyticsBroadcaster,
         }
         router.discardDisplayedCampaign()
     }
+
+    private func flushEventBuffer(void: Bool) {
+        if !void {
+            self.eventBuffer.forEach {
```

Comment:
Is `self.` necessary in this function?

Reply:
not necessary. Copy&paste issue. The same goes for other `self.` in this method